.golangci.yml

```diff
@@ -2,21 +2,21 @@ linters:
   enable:
     - asciicheck
     - bodyclose
-    - depguard
+    # - depguard
```

Comment:
[question] Upstream v0.37.2 doesn't disable depguard, revive, and nakedret so why does our fork need to disable them? If we do disable them, should we create a follow-up GH issue to re-enable them?

Ref: https://github.com/cometbft/cometbft/blob/fe45483be36ebfea7e172a3ad949e8fe09a8fd95/.golangci.yml#L1-L27

Reply:
Yup, these were the most noisy linters and I already think the diff in this PR is too high so I would prefer to focus on them in a separate PR